Doc/using/unix.rst

```diff
@@ -42,20 +42,28 @@ Installing IDLE
 
 In some cases, IDLE might not be included in your Python installation.
 
-* For Debian and Ubuntu users::
+* For Debian and Ubuntu users:
+
+.. code-block:: shell
 
    sudo apt update
    sudo apt install idle
 
-* For Fedora, RHEL, and CentOS users::
+* For Fedora, RHEL, and CentOS users:
+
+.. code-block:: shell
 
    sudo dnf install python3-idle
 
-* For SUSE and OpenSUSE users::
+* For SUSE and OpenSUSE users:
+
+.. code-block:: shell
 
    sudo zypper in python3-idle
```

Comment:
As I said, this will still be highlighted as `in` is a valid shell keyword :( so let's use `install`.

```suggestion
   sudo zypper install python3-idle
```

Reply:
what if we use shell-session lexer?

Reply:
The shell-session lexer seems to work but if we put a prompt (say `$`) in front of the command, it doesn't work. So I'm not sure what would be the best:

![image](https://github.com/user-attachments/assets/6abd78e6-a0f6-4c63-851e-30691b07aad9)

@hugovk what do you recommend here? I think it's not the only place where we're probably mixing lexers but I think  it's better if we don't highlight `in` here. But using `shell-session` without a prompt looks a bit... weird =/

Reply:
Let's use `install` like in the Zypper docs:

https://documentation.suse.com/smart/systems-management/html/concept-zypper/index.html#general-usage-zypper